content/pages/sprint/2021/11late-pycht.rst

```diff
@@ -0,0 +1,71 @@
+=================================================
+Scisprint 2021 Late November: Python ZH TW Doc
```

Comment:
Can you make it explicitly that this is online?  There is an in-person sprint being planned on a close date.

Reply:
@yungyuc you mean like this title?

```markdown
=================================================
(Online) Scisprint 2021 Late November: Python ZH TW Doc
=================================================

...
```

Reply:
> @yungyuc you mean like this title?
> 
> 
> 
> ```markdown
> 
> =================================================
> 
> (Online) Scisprint 2021 Late November: Python ZH TW Doc
> 
> =================================================
> 
> 
> 
> ...
> 
> ```

Yes.

BTW there will be an in-person sprint hosted on the Saturday: https://discord.com/channels/730297880140578906/730393342818648204/905423996550529064

I will be prepping for a PR for it.  FYI.

Reply:
See #88

Reply:
Okay I'll merged #88  to this one